tiled/catalog/adapter.py

```diff
@@ -938,9 +938,18 @@ def _prepare_structure(structure_family, structure):
 
 def binary_op(query, tree, operation):
     dialect_name = tree.engine.url.get_dialect().name
-    attr = orm.Node.metadata_[query.key.split(".")]
+    keys = query.key.split(".")
+    attr = orm.Node.metadata_[keys]
     if dialect_name == "sqlite":
         condition = operation(_get_value(attr, type(query.value)), query.value)
+    # specific case where GIN optomized index can be used to speed up POSTGRES equals queries
+    elif dialect_name == "postgresql" and operation == operator.eq:
```

Comment:
```suggestion
    elif (dialect_name == "postgresql") and (operation == operator.eq):
```
No change in behavior, just a preference for parens when we have multiple conditions, for readability.